perfkitbenchmarker/providers/openstack/flags.py

```diff
@@ -58,6 +58,9 @@
 flags.DEFINE_integer('openstack_volume_size', None,
                      'Size of the volume (GB)')
 
+flags.DEFINE_string('openstack_image_username', 'ubuntu',
+                    'Ssh username for cloud image. Defaults to ubuntu')
```

Comment:
The default will be printed by the help formatter. Could you remove that section?